prusti-specs/src/lib.rs

```diff
@@ -60,6 +60,17 @@ pub fn rewrite_prusti_attributes(
     // Collect the remaining Prusti attributes, removing them from `item`.
     prusti_attributes.extend(extract_prusti_attributes(&mut item));
 
+    // make sure to also update the check in the #[predicate] handling method
+    if prusti_attributes
+        .iter()
+        .any(|(ak, _)| ak == &SpecAttributeKind::Predicate)
+    {
+        return syn::Error::new(
+            item.span(),
+            "`#[predicate]` is incompatible with other prusti attributes",
```

Comment:
```suggestion
            "`#[predicate]` is incompatible with other Prusti attributes",
```

(Unsure if we are consistent about this.)